src/components/UserForm.tsx

```diff
@@ -1,28 +1,66 @@
-import React from 'react';
-import { Container, Title, TextInput } from '@mantine/core';
+import React, { useState, useContext } from 'react';
+import { Container, Title, TextInput, Button, Group, Header } from '@mantine/core';
 import { useStateWithLocalStorage } from './UseStateWithLocalStorage';
+import { WeatherContext } from './WeatherComponent';
+import { MdWbSunny } from 'react-icons/md';
+import { BsFillCloudSnowFill } from 'react-icons/bs';
+import { IoIosPartlySunny } from 'react-icons/io';
 
 const UserForm = () => {
   const [inputValue, setInputValue] = useStateWithLocalStorage('', 'form');
+  const [show, setShow] = useState(true);
+  const weatherIcon = useContext(WeatherContext);
   function handleChange(event: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>) {
     setInputValue(() => ({
       [event.target.name]: event.target.value,
     }));
   }
+
   return (
-    <Container>
-      <Title order={2}>Welcome {inputValue.name ? inputValue.name : ''}</Title>
+    <Header height={56} mb={20}>
+      <Container
+        style={{
```

Comment:
Look into `createStyles` in mantine to handle this sort of thing

Reply:
I've tried to go about it if the button is clicked hide button and show title but am not sure if it will give me the correct result. currently title is hidden which is good but the button only displays none while the mouse is held down.
Could you please have a look:
`const useStyles = createStyles((theme) => ({
  title: {
    visibility: 'hidden',
  },

  button: {
    color: theme.white,
    backgroundColor: theme.colors.blue[6],
    border: 0,
    borderRadius: theme.radius.md,
    padding: `${theme.spacing.sm}px ${theme.spacing.lg}px`,
    cursor: 'pointer',
    margin: theme.spacing.md,

    
    '&:active': {
      display: 'none',
      title: {
        visibility: 'visible',
      },
 ```
   },
  },
```
}));`

Reply:
I woudln't be using CSS for this sort of this. I would just conditionally render the button and title when is it needed

Reply:
Just looking at the comment again. Sorry If i confused you, I was referencing the `style` tag stating that you should replace that with `createStyles` not the component itself